src/docs/product/accounts/pricing.mdx

```diff
@@ -58,300 +54,107 @@ You have two choices for setting your on-demand budget strategy:
+#### Attachments Pricing
 
-All additional events are rejected.
+| Total Attachments Volume | Team/Business Prepaid | Team/Business On-Demand |
+|---|---|---|
+| >25 - 100 (GB) | $0.24 | $0.312 |
+| >100 - 200 (GB) | $0.24 | $0.312 |
+| >200 - 10k (GB) | $0.23 | $0.299 |
 
-#### If I raise my on-demand budget mid-month, when will my organization start accepting events again?
+<!-- prettier-ignore-end -->
 
-Your organization will start accepting additional events as soon as your new budget is applied. We guarantee new budgets will be applied and effective within 24 hours. However, in most cases your organization will start accepting events within minutes.
+You can buy prepaid volume in increments as shown in the table below:
 
-#### If I lower my on-demand budget mid-month below this month’s existing bill, when will the new budget take effect? What will my on-demand bill be?
+<!-- prettier-ignore-start -->
 
-We guarantee your new, lowered on-demand budget will be applied within 24 hours. In the meantime, the old on-demand budget will remain in effect. However, in most cases, the new budget will be applied within minutes.
+<!-- Don't remove spaces from first column header or number ranges in column will break onto two lines making them hard to read -->
 
-After the new budget is in effect, all additional events will be rejected and no additional on-demand capacity will be added. At end of billing month, you will be charged for any on-demand capacity consumed.
+| Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |
+|---|---|---|---|---|---|
+| >50k-100k | 50k | >100k-250k | 150k | >25 - 100 (GB) | 25 (GB) |
+| >100k-500k | 100k | >250k-500k | 250k | >100 - 10k (GB) | 100 (GB) |
+| >500k-2M | 500k | >500k-5M | 500k |  |  |
+| >2M-20M | 1M | >5M-10M | 1M |  |  |
+| >20M-50M | 2M | >10M-20M | 2M |  |  |
 
-#### If I want to cancel monthly billing, what happens?
+<!-- prettier-ignore-end -->
 
-Your plan will continue until the end of the current billing cycle. After this, you’ll revert to the base developer plan, which includes a limited number of events and team members.
+By contrast, on-demand purchasing isn't restricted by any increments since you set a dollar amount for your budget, rather than a certain amount of data. So if you want to purchase an amount of prepaid volume data that falls between increments, you should set that up as on-demand budget.
 
-#### Where can I find my previous invoices/receipts?
+The slider on the [pricing](https://sentry.io/pricing/) page allows you to choose the prepaid volume you'd like for each type of event - errors, transactions, or attachments storage to see what you'd pay.
+
+If your projects have a high volume of events, you can control how many errors Sentry receives. For more information on how to manage your monthly spending, check out our [Quota Management documentation](/product/accounts/quotas/).
```

Comment:
Nit: I'm a little confused whether "events" refers to attachments or if the term "data" should be used instead. I'm seeing many examples of "events" referring to attachments in this doc, but that's not consistent with the definition in `key-terms`.

Reply:
Events is meant to = errors and transactions, while data encompasses all the possible categories. I'll do a sweep of this doc again to make sure I've caught all the mentions.

Reply:
I've gone through and changed any instances of the word "events" to either "events and attachments" or "data" depending on which made more sense.